### PR TITLE
Completed Assignment 2.

### PR DESCRIPTION
**Pull request by delta-plus**
This is Joe Grassl. I'll try to get the next one in a bit sooner!

### CODE REVIEW

File: SecretSanta/src/SecretSanta.Web/Views/Gifts/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@model List<GiftViewModel>
+
+@foreach(var gift in Model) {
+  <div>
+    @gift.Title - @gift.Description
+    <br>
+    Link: @gift.URL
+    <br>
+    Recipient ID: @gift.UserID
```
Comment (Fervidusletum) [inline SecretSanta/src/SecretSanta.Web/Views/Gifts/Index.cshtml:18]: You should present the name associated with the userid to the user, as the userid doesn't have much meaning to them.  
`Recipient ID: @TestData.Users[gift.UserID].FullName`

File: SecretSanta/src/SecretSanta.Web/Views/Groups/Create.cshtml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+@model GroupViewModel
+
```
Comment (Fervidusletum) [inline SecretSanta/src/SecretSanta.Web/Views/Groups/Create.cshtml:2]: You can condense most of the create and edit views into a single template.
[Example here](https://github.com/Fervidusletum/EWU-CSCD379-2021-Spring/tree/Assignment2/SecretSanta/src/SecretSanta.Web/Views/Groups)

**Review by Fervidusletum — APPROVED** [review]
- Create CRUD pages for the following components
- Users
- - A user only needs to have a First and Last name: ✔
- Groups
- - A group will only need to hold the name of the group: ✔
- Gifts
- - A gift should consist of the following information
- - Title: ✔
- - Description: ✔
- - Url: ✔
- - Priority (just a number for holding "order of importance"): ✔
- - The gift should hold a reference to the User it belongs to: ✔
- A Home page that can be used as just a standard landing page for the application: ✔
- Each page should have the ability to navigate between components: ✔

Looks good!

File: SecretSanta/src/SecretSanta.Web/Views/Groups/Create.cshtml (hunk 2 above)
Comment (delta-plus) [reply SecretSanta/src/SecretSanta.Web/Views/Groups/Create.cshtml:2]: All good points. Thanks for the feedback!

**Review by breaman — COMMENTED** [review]
Overall, nice work on the assignment. As your peer pointed out, could have saved yourself some duplicate effort by combining a lot of the functionality for the Create/Edit pages, but wasn't a necessity on this assignment, something to think about for future, though.

**Pull request closed without merging**